pandana/tests/test_pandana.py

```diff
@@ -52,6 +52,63 @@ def test_create_network(sample_osm):
     pass
 
 
+def test_agg_variables_accuracy(sample_osm):
+    net = sample_osm
+
+    # test accuracy compared to pandas functions
+
+    net.set(pd.Series(net.node_ids))
+    s = net.aggregate(10000, type="COUNT")
+    # not all the nodes in the sample network are connected
+    # get the nodes in the largest connected subgraph
+    # from printing the result out I know the largest subgraph has
+    # 477 nodes in the sample data
+    connected_nodes = s[s == 477].index.values
+
+    ssize = 50
+    r = random_data(ssize)
+    nodes = pd.Series(np.random.choice(connected_nodes, ssize))
+    net.set(nodes, variable=r)
+
+    s = net.aggregate(100000, type="COUNT").loc[connected_nodes]
+    assert s.unique().size == 1
+    assert s.iloc[0] == 50
+
+    s = net.aggregate(100000, type="AVE").loc[connected_nodes]
+    assert s.describe()['std'] < .01  # assert almost equal
+    assert np.isclose(s.mean(), r.mean(), atol=1e-3)
+
+    s = net.aggregate(100000, type="MIN").loc[connected_nodes]
+    assert s.describe()['std'] < .01  # assert almost equal
+    assert np.isclose(s.mean(), r.min(), atol=1e-3)
+
+    s = net.aggregate(100000, type="MAX").loc[connected_nodes]
+    assert s.describe()['std'] < .01  # assert almost equal
+    assert np.isclose(s.mean(), r.max(), atol=1e-3)
+
+    r.sort()
+
+    s = net.aggregate(100000, type="MEDIAN").loc[connected_nodes]
+    assert s.describe()['std'] < .01  # assert almost equal
+    assert np.isclose(s.mean(), r.iloc[25], atol=1e-2)
```

Comment:
For testing you can use the NumPy [`assert_allclose`](http://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_allclose.html#numpy.testing.assert_allclose) function. The main benefit there is better output on failures.